client/client.go

```diff
@@ -1870,7 +1870,15 @@ DISCOLOOP:
 func (c *Client) emitStats() {
 	// Assign labels directly before emitting stats so the information expected
 	// is ready
-	c.baseLabels = []metrics.Label{{Name: "node_id", Value: c.NodeID()}, {Name: "datacenter", Value: c.Datacenter()}}
+	var emittedNodeClass string
+	if emittedNodeClass = c.config.Node.NodeClass; emittedNodeClass == "" {
+		emittedNodeClass = "none"
+	}
+	c.baseLabels = []metrics.Label{
```

Comment:
Can you update to

```
c.baseLabels = []metrics.Label{{Name: "node_id", Value: c.NodeID()}, {Name: "datacenter", Value: c.Datacenter()}}
if node := c.Node(); node.NodeClass != "" {
  c.baseLabels = append(c.baseLabels, {Name: "node_class", Value: node.NodeClass})
}
```

Reply:
sure! original idea was to add "none" like it's listed in CLI `nomad node-status`

Reply:
I like `none` idea - having a sparse field for `class` will be confusing :)

Reply:
we will not add the flag to report "none" client class, right?

Reply:
Problem is all possible strings you could add are going to be valid possible node classes

Reply:
@dadgar while working on nomad monitoring tutorial I ran into a problem with grouping nodes by class when node class is not emitted from not nodes with no class configured, so I reverted the code to my previous version when none is emitted from not tagged nodes. 
The issue of all possible strings are going to be valid can be addressed on config parsing stage.
Let me know if you want me to add some restrictions there.